### ISSUE TITLE
 training error

### ISSUE THREAD

**Issue by sanggehouzhihoujue** [weiliu89/caffe#729]
I0904 16:13:26.266971 89932 solver.cpp:243] Iteration 760, loss = 7.36008
I0904 16:13:26.267086 89932 solver.cpp:259]     Train net output #0: mbox_loss = 6.71542 (* 1 = 6.71542 loss)
I0904 16:13:26.267107 89932 sgd_solver.cpp:138] Iteration 760, lr = 0.00025
I0904 16:13:26.840231 89932 solver.cpp:243] Iteration 770, loss = 7.88897
I0904 16:13:26.840288 89932 solver.cpp:259]     Train net output #0: mbox_loss = 6.8737 (* 1 = 6.8737 loss)
I0904 16:13:26.840306 89932 sgd_solver.cpp:138] Iteration 770, lr = 0.00025
caffe: malloc.c:2374: sysmalloc: Assertion `(old_top == (((mbinptr) (((char *) &((av)->bins[((1) - 1) * 2])) - __builtin_offsetof (struct malloc_chunk, fd)))) && old_size == 0) || ((unsigned long) (old_size) >= (unsigned long)((((__builtin_offsetof (struct malloc_chunk, fd_nextsize))+((2 *(sizeof(size_t))) - 1)) & ~((2 *(sizeof(size_t))) - 1))) && ((old_top)->size & 0x1) && ((unsigned long) old_end & pagemask) == 0)' failed.

@weiliu89  When I trained my own dataset, the error occured.
Can you help me?

**Comment by RoseLii** [comment]
please help me .
I download VOCtrainval_06-Nov-2007，but when i running ./data/VOC0712/create_list.sh have error that
y@y-System-Product-Name:~/2T/wangxingli/caffe$ ./data/VOC0712/create_list.sh
Create list for VOC2007 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2012 trainval...
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2012/ImageSets/Main/trainval.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/trainval_img.txt: No such file or directory
Create list for VOC2007 test...
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
cp: cannot stat '$/data/VOCdevkit/VOC2007/ImageSets/Main/test.txt': No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
sed: can't read /home/y/2T/wangxingli/caffe/data/VOC0712/test_label.txt: No such file or directory
paste: /home/y/2T/wangxingli/caffe/data/VOC0712/test_img.txt: No such file or directory
I1025 21:36:51.506589 1277 get_image_size.cpp:61] A total of 0 images.
Why is this wrong?

**Comment by sanggehouzhihoujue** [comment]
@RoseLii  Are you sure the path of the database is right?

**Comment by ZhenpengChenCode** [comment]
Hi, @sanggehouzhihoujue , can you tell me how you fixed the bug? I encountered the same problem.
